raiden/storage/sqlite.py

```diff
@@ -1057,23 +1041,27 @@ def _get_query_with_values(self,
                                                             from_date,
                                                             to_date,
                                                             limit,
-                                                            offset,
-                                                            or_conditional)
+                                                            offset)
 
         return query, tuple_for_execute
 
-    @staticmethod
-    def _get_query_for_node_address(node_address_label, or_contiional):
+    @classmethod
+    def _get_query_for_node_addresses(cls, target_address, initiator_address, our_address):
+        if not target_address and not initiator_address:
+            return ""
+        nodes_query_parts = []
+        if target_address:
+            nodes_query_parts.append(cls._get_query_for_node_address("target", target_address, our_address))
+        if initiator_address:
+            nodes_query_parts.append(cls._get_query_for_node_address("initiator", initiator_address, our_address))
+        return " AND " + " AND ".join(nodes_query_parts)
 
-        result = " AND json_extract(state_events.data, '$.{}') = ? "
-
-        if or_contiional:
-            result = " OR json_extract(state_events.data, '$.{}') = ?  " \
-                     " AND json_extract(state_events.data, '$.token_network_identifier') = ?"
```

Comment:
i'm not sure why or if the token network ID mattered for this query. do you know? maybe we should reflect on this to make sure we aren't making any mistakes

Reply:
The token network is two differentes places.

The code conditionally used an OR, because initator/target field could be missing, and they had to set the token network again to prevent using another token network with the same partner.

We fixed it in another way (using a null check), so it's no longer needed